src/components/PrizePage/Prizes.js

```diff
@@ -81,35 +121,66 @@ export default function Prizes() {
 	const hideImg = useMediaQuery(theme.breakpoints.down('xs'));
 
 	return (
-		<Container maxWidth='md'>
-			<TableContainer>
-				<Table aria-label="prize table">
-					<TableHead>
-						<TableRow>
-							{ hideImg ? null : <TableCell></TableCell> }
-							<TableCell align='center' className={classes.header}>Track</TableCell>
-							<TableCell align='center' className={classes.header}>Prize</TableCell>
-						</TableRow>
-					</TableHead>
-					<TableBody>
-						{allPrizes.map(row =>
-							<TableRow key={row.track}>
-								{ hideImg ?
-									null :
-									<TableCell align='center' width='20%'>
-										<img src={row.img} height='60px'/>
+		<>
+			<Container maxWidth='md'>
+				<Grid maxWidth='md' container spacing={3} className={classes.prizeGrid}>
+					{allPrizes.map(prize =>
+						<Grid item xs={12} sm={4} key={prize.track}>
+							<Box className={classes.prizeBox}>
+								<Box className={classes.imageBox}>
+									<img src={prize.img} height='60px'/>
+								</Box>
+								<Box className={classes.prizeCard}>
+									<Typography variant='h6' component='h3'>
+										{prize.track}
+									</Typography>
+									<Box paddingLeft='20%' paddingRight='20%'>
+										<Squiggle />
+									</Box>
+									<Box className={classes.prizeDescription}>
+										<Typography style={{ color: '#a1a1a1', fontSize: '14px' }}>
+											PRIZE
```

Comment:
nit: don't capitalize in the source code; instead, use https://github.com/uclaacm/hoth.uclaacm.com/blob/483d741f68a8af10631b7377c656dda701a19ed7/src/components/shared-style.js#L3-L11 Example: https://github.com/uclaacm/hoth.uclaacm.com/blob/decae72cbbc2948ee570177d5813ae415f29698d/src/components/SchedulePage/Event.js#L25